packages/core/lib/commands/test/index.js

```diff
@@ -100,8 +100,8 @@ const command = {
         description: "Suppress all output except for test runner output."
       },
       {
-        option: "--bail",
-        description: "Bail after first test failure.  Alias: -b"
+        option: "--bail|-b",
```

Comment:
Are we ok with making this the standard way to document aliases in the help? Currently in some places they are listed at the end of the description. See [debug's description for `--fetch-external`](https://github.com/trufflesuite/truffle/blob/develop/packages/core/lib/commands/debug.js#L44). I would propose we change all instances of that to this format. Changing these will also prevent Truffle from warning the user for these aliases now that the algorithm for determining valid options has been updated. See the changes in this PR for `packages/core/lib/command.js`.

Reply:
I prefer this location but wouldn't mind replacing the pipe with a comma: `--bail, b`

Reply:
I'm not dead set against the `,` but prefer the pipe. What do other command line tools use for this kind of a situation? Should we take a vote?

Reply:
I've seen cli option aliases listed with both commas or spaces. I don't think it matters in this case. it looks odd and I'm willing to look past :)

Reply:
Well let's implement whatever is common practice. I'll go ahead and change this to `,`.

Reply:
Well git does space pipe like this 
<img width="854" alt="Screen Shot 2021-07-16 at 11 38 48 AM" src="https://user-images.githubusercontent.com/14827965/125973331-2c214d6d-b3ac-4ddb-af74-d3963ea4c53b.png">